src/dbnode/server/server.go

```diff
@@ -21,6 +21,7 @@
 package server
 
 import (
+	stdlib "context"
```

Comment:
nit: need a name that's clearer about defining the import "stdcontext" or "stdlibcontext" or "context" instead of "stdlib" please